extensions/amp-ad-exit/0.1/amp-ad-exit.js

```diff
@@ -100,8 +103,20 @@ export class AmpAdExit extends AMP.BaseElement {
     if (target.vars) {
       for (const customVar in target.vars) {
         if (customVar[0] == '_') {
-          vars[customVar] = () =>
-              args[customVar] || target.vars[customVar].defaultValue;
+          const vals =
+              /** @type {./config.Variable} */ (target.vars[customVar]);
+          vars[customVar] = () => {
+            if (vals.vendorAnalyticsSource) {
+              const map = IframeTransportResponseMap.get(this.getAmpDoc(),
+                  vals.vendorAnalyticsSource,
+                  /** @type {string} */ (this.win.document.baseURI));
+              if (map && vals.vendorAnalyticsResponseKey &&
```

Comment:
What happens if map[vals.vendorAnalyticsResponseKey] is a falsy value (e.g. 0, false)?  Wouldn't that cause this predicate to return false and so the value wouldn't be macro-ed in?